demo/demo_instructions.rst

```diff
@@ -25,10 +25,21 @@ There are three sites (Jacksonville, UCSD, Omaha) currently available on IARPA's
+The only fields that need to be filled out in this file are ``gsd`` and ``cuda``. The first is the desired ground sample distance (default .25), and the second is whether or not CUDA is available. If Danesfield is being run without images, the gsd field can be ignored. 
+The other configuration file is for VisSat Satellite Stereo, and it must be named ``<region>_config.json``. An example of this file can be found at https://github.com/Kitware/Danesfield/blob/master/input.ini. The ``bounding_box`` fields are related to the area that you want to reconstruct with Danesfield. They include the UTM zone, easting, and northing of the bounding box's upper left corner, as well as its width and height in meters. The ``steps_to_run`` fields should be as they appear in the example. The final fields, ``alt_min`` and ``alt_max``, are the approximate minimum and maximum altitudes present in the area of interest. This file is not necessary if Danesfield is being run without images.
+
+The two example configuration files linked above should be copied into the configuration directory and renamed to the required names. Edit the necessary fields and the rest of the fields will be filled in by the demo script. 
+
+Three example configuration files, one for each available region, come with this demo. 
+
 Running Danesfield
 ------------------
 
-After downloading the Docker container, imagery, and models, the script can run the Danesfield container on the source imagery. This is achieved with the ``--run`` option. When running Danesfield, one must specify the location of the source imagery, model files, an output directory, and configuration files. Use the ``--model_path`` argument to provide the directory where the necessary model files are located. Use the ``--config_path`` argument to provide the directory where the necessary configuration files are located. Finally, use the ``--out_path`` argument to provide where the directory where Danesfield should save all its output. If none of these paths are specified, then these directories will be created within the current directory as a default. 
+After downloading the Docker container, imagery, and models, the demo script can run the Danesfield container on the source imagery. This is achieved with the ``--run`` option. When running Danesfield, one must specify the location of the source imagery, model files, an output directory, and configuration files. Use the ``--model_path`` argument to provide the directory where the necessary model files are located. Use the ``--config_path`` argument to provide the directory where the necessary configuration files are located. Finally, use the ``--out_path`` argument to provide where the directory where Danesfield should save all its output. If none of these paths are specified, then these directories will be created within the current directory as a default. 
```

Comment:
Consider rephrasing: [``--out_path`` argument to provide where the directory where Danesfield should save all its output] => [``--out_path`` argument to provide the directory where Danesfield should save all its output]